libs/api/domains/islykill/src/lib/islykill.type.ts

```diff
@@ -0,0 +1,25 @@
+export type IslykillErrorResult = {
+  message: string
+  code: number
+  invalidFields: string[]
+}
+
+export type IslykillUpdateResponse = {
+  nationalId: string
+  valid: boolean
+  error?: IslykillErrorResult
+}
+
+export type IslykillSettings = {
+  nationalId: string
+  email?: string
+  mobile?: string
+  bankInfo?: string
+  lastLogin?: string
+  nextLastLogin?: string
+  lastPassChange?: string
+  canNudge?: boolean
+  onlyCert?: boolean
+  nudgeLastAsked?: string
+  noUserFound?: boolean
+}
```

Comment:
Whenever possible, interfaces should be used instead of types.

Feels like these types are redundant. Generally our domain services work with our dto types directly.

Reply:
Yes, that makes sense.
Most of these redundant types were put there initially before it was clear which would be used.
Cleaning this up 🧹